specification/sigma-appendix-modifiers.md

```diff
@@ -58,7 +58,6 @@ The modifiers listed in this section can only be applied to string values.
     - Anchors: `^`, `$`.
     - Quantifiers: `*`, `+`, `?`, `{n,m}`.
     - Character Classes: [a-z], [^a-z].
-    - Grouping and Capturing: `()`.
```

Comment:
I would only remove the "capturing" which means that a value contained in a group can be referenced later in the regular expression like `id=([a-z]+), foo=\1`. While grouping itself should be available in all regular expression variants, e.g. `(repeat[0-9])+`.